test/integration/tests/post_Update_PBA.ts

```diff
@@ -0,0 +1,12 @@
+import { generatePOSTAPIRequest } from './utils';
+const should = require('chai').should();
+
+suite('Approve Org -> PUT Update PBA numbers', function() {
+  this.timeout(50000);
+  const payload = {
+  paymentAccounts: [`PBA1${Math.floor(100000 + Math.random() * 900000)}`, `PBA2${Math.floor(100000 + Math.random() * 900000)}`],
+  orgId: 'BOHOBFK',
+  };
+  test('PUT Update PBA numbers', () => generatePOSTAPIRequest ('PUT', '/api/updatePba', payload));
```

Comment:
missing assertion statement in test.

Reply:
not required for PBA as they randomly generated jsut validating the request